### ISSUE TITLE
[FEATURE] multiple events for a habit per day

### ISSUE THREAD

**Issue by khimaros** [FriesI23/mhabit#117]
**Describe the solution you'd like**
ability to set a habit to be done eg. "3 times per 1 day"

**Comment by zatevakhin** [comment]
@Miaaaim I wanted to create the issue for "Multiple reminders in one day", and found this issue and think this can be related. Is it appropriate to request this feature here or better create another issue for this?

**Comment by khimaros** [comment]
@Miaaaim i don't believe this solves the use case. how can i record multiple events throughout the day?

**Comment by khimaros** [comment]
i see, the value can be changed by press and hold. a bit higher friction than i would imagine. any way to make it record one increment per press?

**Comment by Miaaaim** [comment]
@zatevakhin You can set individual reminders for each habit. Hopefully, this resolves your issue.

**Comment by Miaaaim** [comment]
> i see, the value can be changed by press and hold. a bit higher friction than i would imagine. any way to make it record one increment per press?

Thanks for your suggestion. Actually we've considered this approach before, but there wasn't a particularly straightforward way to implement it. Currently, you need to long-press and then click to increment the value in the pop-up. However, if we come up with a better method in the future, we'll optimize it.
